modules/accredited_representative_portal/spec/sidekiq/accredited_representative_portal/redact_power_of_attorney_requests_job_spec.rb

```diff
@@ -0,0 +1,511 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+require_relative '../../spec_helper'
+
+Sidekiq::Testing.fake!
+
+# rubocop:disable Metrics/ModuleLength
+module AccreditedRepresentativePortal
+  RSpec.describe RedactPowerOfAttorneyRequestsJob, type: :job do
+    # Helper to add redactable data to a submission
+    def add_redactable_submission_data(submission)
+      # Disable validation skip check: Directly setting ciphertext for test setup.
+      # rubocop:disable Rails/SkipsModelValidations
```

Comment:
I see a number of these in this file. Why is it necessary to skip the model validation?

Reply:
Oh yeah. I'm doing this because we have those fields that are being redacted required on creation or updates. But the redaction means they are deleted which will not pass validation.